src/H5CX.c

```diff
@@ -449,6 +449,9 @@ static H5CX_node_t *H5CX__pop_common(bool update_dxpl_props);
 /* Package Variables */
 /*********************/
 
+/* Package initialization variable */
+bool H5_PKG_INIT_VAR = false;
```

Comment:
`H5_PKG_INIT_VAR` and `H5_MY_PKG_INIT` are easily confused names. Maybe something like `H5_PKG_IS_INITIALIZED` and `H5_MY_PKG_USES_INIT`?

Reply:
I'm fine either way.  If you want to submit a followup PR, I'd be good with that.